client/components/Dashboard.js

```diff
@@ -0,0 +1,181 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { fetchMeetups } from '../store';
+import { Redirect } from 'react-router-dom';
+// import * as _ from 'lodash';
+import axios from 'axios';
+// import {daysInMonth} from '../../utils';
+
+class Dashboard extends Component {
+    constructor(props){
+        super();
+        this.state = {
+        	userApi: {},
+        	//put all of this user's meetups in the local state
+        	meetupIdArray: [],
+          meetupIncludeUserArray: []
+
+        };
+        // this.handleSubmit = this.handleSubmit.bind(this);
+        // this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleSubmit(event) {
+    }
+
+    handleChange(event) {
+    }
+    
+    componentDidMount(props){
+    	const userId = this.props.user.id
+    	//bad to do this directly in here instead of putting on store?
+    	axios.get(`/api/users/${userId}`)
+    	.then(res => {
+    		return res.data})
+    	.then(result => {
+    		this.setState({userApi: result})
+    	})
+
+      //not using anymore. delete at end of view.
+      // const meetupsThunk = fetchMeetups(userId)
+      // store.dispatch(meetupsThunk)
+    	
+      //given user id, give me a list of meetup ids and a list of meetups that include users in that meetup
```

Comment:
Same here - really makes more sense to do in the store as a thunk

Reply:
agreed.